shard-operator/src/main/kubernetes/minikube.yml

```diff
@@ -6,16 +6,26 @@ rules:
   - apiGroups:
```

Comment:
@vaibhavjainwiz @r00ta Super-basic question to help me understand better: What are these files used for and how are they generated? 

They look the same as  the `ClusterRole` that we have in `kustomize/base-openshift/shard/cluster-role.yaml` so is our current process to generate these somehow and then copy them into the `kustomize` directory?

cheers,
Rob

Reply:
files in `shard-operator/src/main/resources/kubernetes` are how we customise the default files generated by the java operator sdk dependency. 
>  copy them into the `kustomize` directory?
I am not sure, How we are maintaining `kustomize` dir. @r00ta Could you please help us with it.

Reply:
yes @robpblake, we just copy clusterRole from `src/main/kubernetes/minikube.yml` to `kustomize`.